dist/mask.js

```diff
@@ -459,10 +462,14 @@ angular.module('ui.mask', [])
                                 return maskCaretMap.indexOf(pos) > -1;
                             }
 
-                            function getCaretPosition(input) {
+                            function getCaretPosition(input, eventWhich) {
                                 if (!input)
                                     return 0;
+                                var isRemoving = eventWhich === 8 || eventWhich === 46;
```

Comment:
If you have a moment could you explain how this is functioning for android? It's not a blocker; i'm just curious :)